lib/shared/units.js

```diff
@@ -42,36 +31,22 @@ const MEGABYTE_TO_BYTE_RATIO = 1e+6
  */
 const MILLISECONDS_IN_A_DAY = 86400000
 
-/**
- * @summary Convert bytes to megabytes
- * @function
- * @public
- *
- * @param {Number} bytes - bytes
- * @returns {Number} megabytes
- *
- * @example
- * const result = units.bytesToMegabytes(7801405440);
- */
-exports.bytesToMegabytes = (bytes) => {
-  return bytes / MEGABYTE_TO_BYTE_RATIO
-}
-
 /**
  * @summary Convert bytes to most appropriate unit string
  * @function
  * @public
  *
  * @param {Number} bytes - bytes
+ * @param {String} [suffix=''] - suffix string e.g. '/s'
  * @returns {String} size and unit string
  *
  * @example
- * const humanReadable = units.bytesToClosestUnit(7801405440);
- * > '7.8 GB'
+ * const humanReadable = units.bytesToClosestUnit(7801405440, '/s');
+ * > '7.8 GB/s'
  */
-exports.bytesToClosestUnit = (bytes) => {
-  if (_.isNumber(bytes)) {
-    return prettyBytes(bytes)
+exports.bytesToClosestUnit = (bytes, suffix = '') => {
+  if (_.isFinite(bytes)) {
+    return prettyBytes(bytes) + suffix
   }
 
   return null
```

Comment:
This method should be consistent in the data type returned;
Let's make this return an empty string in this case – that's what the flash state tests are apparently expecting anyways. Which bears the question: Why aren't they failing?

Reply:
> Which bears the question: Why aren't they failing?

Perhaps that implies that the `bytesToClosestUnit` function needs more comprehensive unit-tests?